news-search/src/components/Navbar.js

```diff
@@ -22,17 +22,20 @@ function NavbarExpand() {
         <Navbar.Toggle aria-controls="responsive-navbar-nav" />
         <Navbar.Collapse id="responsive-navbar-nav">
           <Nav className="me-auto">
-            <Nav.Link as={Link} to='/'>Home</Nav.Link>
+            <Nav.Link as={Link} to='/'><div onClick={allTopics}>Home</div></Nav.Link>
             <Nav.Link as={Link} to='/'>About</Nav.Link>
             <NavDropdown title="Topics" id="collasible-nav-dropdown">
-              <NavDropdown.Item as={Link} to='/global-warming'>Global Warming</NavDropdown.Item>
-
-              <NavDropdown.Item as={Link} to='/bitcoin'>
-                Bitcoin
+              <NavDropdown.Item>
```

Comment:
here as well, maybe including logic like a loop for creating these elements over the topics would work?